BigQuery/15_convert_schema_for_labeling.sql

```diff
@@ -0,0 +1,50 @@
+SELECT 
+subject_id as id,
+row_number() over (partition by subject_id, hadm_id, stay_id order by chart_hour asc) as ICULOS,
+admittime as HospAdmTime,
+0 as Unit1,
+1 as Unit2,
```

Comment:
Do we know that the unit is definitely the surgical/medical unit? If not, just leave it n/a.

Reply:
I had assumed that Unit 2 was to represent that this data came from MIMIC (if we ever combined it with other data sources) - I should have confirmed with RJ previously on why we're putting in the default values for these fields

Reply:
Ok. I'm ok with wrapping up this PR and adding it as a `TODO` for later - whatever you like.